chat.go

```diff
@@ -29,13 +30,18 @@ func main() {
 	}
 
 	if err := checkServer(client, params); err != nil {
-		log.Fatalf("Connecting to server failed: %v", err)
+		err = fmt.Errorf("%v\n\nUnable to chat. Make sure the server has started successfully.\n", err)
+		fmt.Fprint(os.Stderr, err)
+		return
```

Comment:
Should have non-zero code:
```suggestion
		os.Exit(1)
```